src/vapor/common/card.vue

```diff
@@ -1,17 +1,19 @@
 <template>
     <article class="vapor-card"
-             :class="theme">
+            :class="theme">
         <img class="vapor-card__image"
-             :src="iconSource"/>
+            alt=""
```

Comment:
I think this should be bound to a new alt property [in this data](https://github.com/vapor/website/blob/219771f3b3eabdb80bd174b39ededee1b6e23730/src/vapor/sponsor.vue#L96).